Tools/jhbuild/jhbuild-minimal.modules

```diff
@@ -171,15 +171,13 @@
             hash="sha256:85803ccbdbdd7a3b03d930187cb055f1353596969c1f92ebec2db839fa4f834a"/>
   </autotools>
 
-  <!-- glib needed to upgrade gdbus-codegen -->
   <meson id="glib"
          mesonargs="-Dlibmount=disabled -Dselinux=disabled">
-    <dependencies>
-      <dep package="libffi"/>
-    </dependencies>
-    <branch module="/sources/glib/2.70/glib-${version}.tar.xz" version="2.70.0"
+    <pkg-config>glib-2.0.pc</pkg-config>
+    <branch module="/sources/glib/2.70/glib-${version}.tar.xz"
```

Comment:
I think module should be 2.71: https://download.gnome.org/sources/glib/2.71/glib-2.71.3.tar.xz